types/chance/index.d.ts

```diff
@@ -224,6 +225,10 @@ declare namespace Chance {
         extensions: string[];
     }
 
+    interface MacOptions {
+        separator: string;
```

Comment:
```suggestion
        separator?: string;
```
It's optional by design, never used in other shape I think
also, does it need network prop?
https://github.com/chancejs/chancejs/blob/dc1a29a0d6963ecf4ca0bb2e69448081d621c877/test/test.misc.js#L172

Reply:
the use of `Partial` makes it so making each individual prop doesn't have to be marked as optional. 

but yea, adding a `networkVersion: boolean` here would be ideal.